nodejs/index.js

```diff
@@ -102,44 +102,45 @@ function validateHandlerDefinition(userHandler, handlerName, moduleName) {
   }
 }
 
-async function importHandler() {
-  const { LAMBDA_TASK_ROOT = '.' } = process.env
-  const { moduleToImport, handlerToWrap } = getHandlerPath()
+let wrappedHandler
+let patchedHandlerPromise
 
+const { LAMBDA_TASK_ROOT = '.' } = process.env
```

Comment:
did you mean to put this stuff in the node 20 PR?

Reply:
This is rebased off of the performance changes, so I'd imagine that the diff would show that branch's change. I could change the target, or roll back that rebase if we prefer.

Reply:
yea. i'd just base this off of the main branch because it's hard to see what's actually changed to support node 20

Reply:
Updated to be based off the current main.